analyses/focal-cn-file-preparation/03-prepare-cn-file.R

```diff
@@ -289,7 +289,7 @@ readr::write_tsv(autosome_annotated_cn,
 if (xy_flag) {
   # Filter to just the X and Y chromosomes
```

Comment:
We can also add here that we filter out the neutral copy number statuses, and the brief explanation for this decision.